README.md

```diff
@@ -2,42 +2,29 @@
 
 Django-Todolist is a todolist web application with the most basic features of most web apps, i.e. accounts/login, API and (somewhat) interactive UI.
 
----
-CSS | [Skeleton](http://getskeleton.com/)
-JS  | [jQuery](https://jquery.com/)
+## Docker Compose Usage
+To run and stop containers with Docker Compose, follow these steps:
 
-## Explore
-Try it out by installing the requirements. (Works only with python >= 3.8, due to Django 4)
+### Running Containers
+1. Navigate to the directory containing the Docker Compose file.
+2. Run the following command to start the containers:
 
-    pip install -r requirements.txt
+    ```bash
+    docker-compose up
+    ```
 
-Create a database schema:
+   This will build and start both the MySQL database and the Todolist app.
 
-    python manage.py migrate
-
-And then start the server (default: http://localhost:8000)
-
-    python manage.py runserver
+### Stopping Containers
+To stop the containers, press `Ctrl + C` in the terminal where `docker-compose up` is running. Alternatively, you can run the following command:
```

Comment:
The alternative way is applicable only if the previous `up` command was executed in detached mode
`docker-compose up -d` And detached mode is even more commonly used.